site/content/en/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -60,7 +60,7 @@ You then need to create `ClusterRole` and `ClusterRoleBinding` for that same (de
 using a command:
 
 ```shell
-kubectl apply -f https://kueue.sigs.k8s.io/examples/admin/cluster-role-and-binding.yaml
+kubectl apply -f https://kueue.sigs.k8s.io/examples/visibility/cluster-role-and-binding.yaml
```

Comment:
I confirmed that this is valid fixes: https://github.com/kubernetes-sigs/kueue/blob/96294275676608ca641afad1e90421c2092c4971/site/static/examples/visibility/cluster-role-and-binding.yaml